### PR TITLE
Profiles form layout position

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@saeedmb8 please +1 

Registration Page: move pin fieldsetto the bottom of page 
screen shot 2017-10-10 at 11 10 58 am

File: iogt/styles/iogt/modules/_m-profiles.scss [hunk 1]
```diff
@@ -23,12 +26,17 @@
     padding: 0;
   }
 }
+label.profiles__terms {
+    color: #666;
```
Comment (nathanbegbie) [inline iogt/styles/iogt/modules/_m-profiles.scss]: Should we use `$victoria` here again?

File: iogt/templates/profiles/register.html [hunk 2]
```diff
@@ -107,14 +100,21 @@ <h4 class="heading heading--subtitle">
...
+    <fieldset>
       {% if form.terms_and_conditions.errors %}
         <span class="error">
           {% trans "Please accept the T&amp;Cs in order to complete the registration" %}
         </span>
       {% endif %}
       {{ form.terms_and_conditions }}
       {% if settings.profiles.UserProfilesSettings.terms_and_conditions %}
-      <a href="{% pageurl settings.profiles.UserProfilesSettings.terms_and_conditions %}" for="id_terms_and_conditions" class="profiles__terms">{% trans "I accept the Terms and Conditions" %}</a>
+      <a href="{% pageurl settings.profiles.UserProfilesSettings.terms_and_conditions %}" for="id_terms_and_conditions" target="_blank" class="profiles__terms">{% trans "I accept the Terms and Conditions" %}</a>
```
Comment (nathanbegbie) [inline iogt/templates/profiles/register.html]: Just for my own knowledge - what does `target="_blank"` do?

File: iogt/templates/profiles/register.html [hunk 3]
```diff
@@ -107,14 +100,21 @@ <h4 class="heading heading--subtitle">
     {% endif %}
 
     <fieldset>
+      <label for="id_password">{% trans "Choose a 4-digit pin" %}</label>
+      {{ form.password.errors}}
+      {{ form.password }}
+      <span class="input-hint">({% trans "e.g. 2086" %})</span>
```
Comment (nathanbegbie) [inline iogt/templates/profiles/register.html]: We could change this to `{% trans "e.g." %} 2086` for max translation reuse

**Comment by nathanbegbie** [comment]
2 small comments, otherwise :+1:

**Pull request merged**